apis/theme/src/index.js

```diff
@@ -76,21 +76,26 @@ export default function theme() {
      * Get the value of a style attribute in the theme by searching in the theme's JSON structure.
      * The search starts at the specified base path and continues upwards until the value is found.
      * If possible it will get the attribute's value using the given path.
+     * When a default value is provided and there is no value of
+     * a style attribute found with that exactly specified path,
+     * the default value will be returned.
      *
      * @param {string} basePath - Base path in the theme's JSON structure to start the search in (specified as a name path separated by dots).
      * @param {string} path - Expected path for the attribute (specified as a name path separated by dots).
      * @param {string} attribute - Name of the style attribute.
-     * @returns {string} The style value
+     * @param {string|null} [defaultValue] - Set a default value if no style value found.
```

Comment:
Do we know the type of this? If the value you are looking for is a number, the defaultValue could be a number

Reply:
It can be 'Opinionated'. I do not think it is good to allow all types and return any types. so here I personally decided that it only accepts string or null.

Reply:
If there are properties that are of other types, we should allow both defaultValue and return to be of those types. If it is only string | null than we can keep this

Reply:
Yes, lets keep this